CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # PayPal Android SDK Release Notes
 
 ## unreleased
-
+* PaymentButtons
+  * Button colors that were deprecated in the `1.3.0` release are being reinstated.
```

Comment:
To avoid merchants having to scroll and figure out what we mean I think we can go with something more explicit like:
```suggestion
  * Undeprecate `PayPalCreditButtonColor.BLACK` and `.DARK_BLUE`
  * Undeprecate `PayPalButtonColor.BLUE`, `.BLACK`, and `.SILVER`
```